instance/tasks.py

```diff
@@ -66,6 +66,7 @@ def watch_pr():
             instance, created = OpenEdXInstance.objects.get_or_create(
                 sub_domain=pr_sub_domain,
                 fork_name=pr.fork_name,
+                github_pr_number=pr.number,
```

Comment:
@smarnach True, it would have to be done carefully in the migrations, to handle potential API call failures. But here your approach sounds good to me - it means that we wouldn't be able to have several sandboxes from a single branch, even if they have multiple PRs (even if they are successive) - but that might actually turn out to be a feature. We'll see.